nautobot_netbox_importer/management/commands/import_netbox_objectchange_json.py

```diff
@@ -0,0 +1,134 @@
+        )
+        parser.add_argument("netbox_version", type=version.parse)
+        parser.add_argument("--dry-run", action="store_true", default=False)
+
+    def map_object_type(self, key, entry, used_error_messages):
+        """Map a ContentType object from Netbox to Nautobot. Returns False is not possible."""
+        app_label, modelname = self.netbox_contenttype_mapping[entry["fields"][key]]
+        try:
+            contenttype_id = self.nautobot_contenttype_mapping[(app_label, modelname)]
+            entry["fields"][key] = ContentType.objects.get(id=contenttype_id)
+        except KeyError:
+            error_message = f"{(app_label, modelname)} key is not present in Nautobot"
+            if error_message not in used_error_messages:
+                used_error_messages.append(error_message)
+                self.logger.warning(error_message)
+            return False
+        return True
+
+    def process_objectchange(self, entry, options, used_error_messages):
+        """Processes one ObjectChange entry (dict) to migrate from Netbox to Nautobot."""
+        if not self.map_object_type("changed_object_type", entry, used_error_messages):
+            return
+
+        if entry["fields"]["related_object_type"]:
+            if not self.map_object_type("related_object_type", entry, used_error_messages):
+                return
+        else:
+            del entry["fields"]["related_object_type"]
+
+        modelname = entry["fields"]["changed_object_type"].model
+        entry["fields"]["changed_object_id"] = netbox_pk_to_nautobot_pk(modelname, entry["fields"]["changed_object_id"])
+        if entry["fields"]["related_object_id"]:
+            modelname = entry["fields"]["related_object_type"].model
+            entry["fields"]["related_object_id"] = netbox_pk_to_nautobot_pk(
+                modelname, entry["fields"]["related_object_id"]
+            )
+
+        entry["fields"]["user"] = User.objects.filter(username=entry["fields"]["user_name"]).first()
+        if not entry["fields"]["user"]:
+            error_message = f'Username {entry["fields"]["user_name"]} not present in DB.'
+            if error_message not in used_error_messages:
+                used_error_messages.append(error_message)
+                self.logger.error(error_message)
+            return
+
+        if not options["dry_run"]:
+            obj = ObjectChange.objects.create(**entry["fields"])
+            obj.time = entry["fields"]["time"]
+            obj.full_clean()
+            obj.save()
+
+    def handle(self, *args, **options):
+        """Handle execution of the import_netbox_json management command."""
+        validate_netbox_version(options["netbox_version"])
+
+        self.logger, _ = initialize_logger(options)
+
+        self.logger.info("Loading NetBox JSON data into memory...", filename={options["json_file"].name})
+        no_objectchange_data = json.load(options["json_file"])
```

Comment:
`netbox-docker`, by default, prepends a few lines of text log output to any `manage.py` command, including `manage.py dumpdata`, which causes the resulting file to not be valid JSON until/unless you manually delete those lines with a text editor. I mention this just to suggest that you add a `try...except ValueError` around each of the `json.load` calls so that if the file isn't valid JSON, you can handle it gracefully with an appropriate error message.

Reply:
thanks for the hint
btw, I believe that the same applies to https://github.com/nautobot/nautobot-plugin-netbox-importer/blob/develop/nautobot_netbox_importer/management/commands/import_netbox_json.py#L132